src/test/end-to-end/tests/admin/user-management-test.ts

```diff
@@ -0,0 +1,80 @@
+import { config as testConfig } from '../../../config';
+import { randomData } from '../../shared/random-data';
+
+Feature('Manage admin user accounts');
+
+const TEST_WORKER_NUMBER = randomData.getRandomNumber(10000000, 99999999);
+const TEST_FIRST_NAME = testConfig.TEST_SUITE_PREFIX + 'FirstName';
+const TEST_LAST_NAME = testConfig.TEST_SUITE_PREFIX + 'Surname';
+const TEST_EMAIL = randomData.getRandomEmailAddress(TEST_WORKER_NUMBER);
+const TEST_ROLE = 'Internal - Administrator - Local';
+const TEST_INVALID_EMAIL = 'invalid_email@test.com';
+
```

Comment:
All test email addresses should end up with either justice,gov.uk or hmcts.net, even the invalid addresses :)